test/permissions.spec.js

```diff
@@ -71,9 +103,9 @@ module.exports.describe = function({testRunner, expect, WEBKIT}) {
         });
       });
       expect(await page.evaluate(() => window['events'])).toEqual(['prompt']);
-      await context.setPermissions(server.EMPTY_PAGE, []);
+      await context.grantPermissions([], { origin: server.EMPTY_PAGE });
```

Comment:
`server.EMPTY_PAGE` is not quite an origin. Do we support urls? I don't see where we convert url to an origin.

Reply:
I think it happens inside the browsers :/